webpack.config.js

```diff
@@ -1,58 +1,64 @@
-var webpack = require('webpack');
-var PROD = process.env.NODE_ENV === 'production';
+/* global __dirname: true */
+const webpack = require('webpack');
+const PROD = process.env.NODE_ENV === 'production';
```

Comment:
Has running `npm start` still worked fine with this change? By default I don't believe webpack is interpreted in ES6. [This might help](http://stackoverflow.com/questions/31903692/how-can-i-use-es6-in-webpack-config-js). Currently I'm actually happy keeping this file in ES5 syntax (rather than getting more tools and setup).

If we're going to convert this file to ES6, then the imports/exports and the function need to have their syntax adjusted.